src/components/menus/LocalDocsMenu.ts

```
@@ -0,0 +1,683 @@
+        }
+    }
+    
+    /**
+     * Attempt to connect to the manager when the editor is ready
+     * This can be called after initialization if the manager wasn't available initially
+     */
+    connectToManager() {
+        if (!this.manager && this.editor && this.editor.innerEditor) {
+            try {
+                const localDocsExt = this.editor.innerEditor.extensionManager.extensions.find(ext => ext.name === 'localDocs');
+                if (localDocsExt && localDocsExt.storage) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)